action/action.yml

```diff
@@ -15,6 +15,9 @@ inputs:
   bindir:
     description: "Optional location of the Flux binary. Will not use sudo if set. Updates System Path."
     required: false
+  token:
+    description: "GitHub Token used to authentication against the API (generally only needed to prevent quota limit errors)"
```

Comment:
Small typo here.
```suggestion
    description: "GitHub Token used to authenticate against the API (generally only needed to prevent quota limit errors)"
```